client/src/components/HelloWorld.vue

```diff
@@ -1,59 +0,0 @@
-<template>
-  <div class="hello">
-    <h1>{{ msg }}</h1>
```

Comment:
remove this from commit

Reply:
Yes, It has been removed in next commit :D .